content/docs/3_reference/3_panel/4_sections/0_pages/reference-panelsection.txt

```diff
@@ -194,6 +194,18 @@ columns:
     value: "{{ page.date.toDate('d.m.Y') }}"
 ```
 
+##### Mobile
+
+Set the `mobile` option to `true` for a column to visible in the mobile view.

```

Comment:
Set the `mobile` option to `true` for a column to be visible in the mobile view.

Reply:
Thank you. You can correct my typos directly 🙂